__tests__/app.test.js

```diff
@@ -63,6 +63,42 @@ describe("GET /api/reviews", () => {
         expect(body.reviews).toBeSortedBy("created_at", { descending: true });
       });
   });
+  test("status - 200, can select a category for the returned reviews", () => {
+    return request(app)
+      .get("/api/reviews?category=dexterity")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        expect(reviews).toBeInstanceOf(Array);
+        body.reviews.forEach((review) => {
+          expect(review.category).toBe("dexterity");
+        });
+      });
+  });
+  test("status - 200, sorts by selected direction", () => {
+    return request(app)
+      .get("/api/reviews?order=asc")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.reviews).toBeSortedBy("created_at", { descending: false });
+      });
+  });
+  test("status - 200, sorts by selected column", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=votes&order=asc")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.reviews).toBeSortedBy("votes");
+      });
+  });
+  test("status - 400, invalid query", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=notAColumn")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid query");
+      });
+  });
```

Comment:
theres a few more errors to handle here. What about a category that doesnt exist? or an order other than asc/desc? and can you prove all three of these queries work together?